scipy/stats/tests/test_continuous.py

```diff
@@ -1272,6 +1273,70 @@ def cdf(self, x, *, c, mu, sigma):
         assert_allclose(X1.icdf(p), X2.icdf(p))
         assert_allclose(X1.iccdf(p), X2.iccdf(p))
 
+    @pytest.mark.parametrize("a", [0.5, 1.0, 2.0, 4.0, 8.0])
+    @pytest.mark.parametrize("b", [0.5, 1.0, 2.0, 4.0, 8.0])
```

Comment:
To save some test time and strengthen the test, maybe:
```suggestion
    @pytest.mark.parametrize("a", [0.5, np.asarray([0.5, 1.0, 2.0, 4.0, 8.0])])
    @pytest.mark.parametrize("b", [0.5, np.asarray([0.5, 1.0, 2.0, 4.0, 8.0])])
```
That way we test with arrays and not just scalars, but we only run the test four times instead of 25.